03/app.js

```diff
@@ -42,10 +42,40 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
+  const title = this[isbn]["title"];
```

Comment:
Warto sprawdzić przed czy `this[isbn]` istnieje tj. czy nie jest różne od `undefined`